### ISSUE TITLE
fix(versions): support link: versions

### ISSUE THREAD

**Issue by bombillazo** [JamieMason/syncpack#114]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `syncpack@8.5.14` for the project I'm working on.

I added this line to my build process since syncpack would fail if a local package was used. We can identify local packages with `link:` or using a local directory in the path.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/syncpack/dist/bin-fix-mismatches/fix-mismatches.js b/node_modules/syncpack/dist/bin-fix-mismatches/fix-mismatches.js
index 2909484..f34cae0 100644
--- a/node_modules/syncpack/dist/bin-fix-mismatches/fix-mismatches.js
+++ b/node_modules/syncpack/dist/bin-fix-mismatches/fix-mismatches.js
@@ -11,6 +11,7 @@ function fixMismatches(ctx) {
         // Set the correct version on each instance.
         invalidGroups.forEach(function (instanceGroup) {
             var nextVersion = instanceGroup.getExpectedVersion();
+            if(!nextVersion || nextVersion.includes('link:') || nextVersion.substring(0,2) === './') return;
             instanceGroup.instances.forEach(function (instance) {
                 return instance.setVersion(nextVersion);
             });
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by JamieMason** [comment]
Thanks a lot @bombillazo that's great

**Comment by bombillazo** [comment]
Some testing may be required but its a start 😁

**Comment by JamieMason** [comment]
Released in [syncpack@10.0.0](https://github.com/JamieMason/syncpack/releases/tag/10.0.0), please let me know if you run into any issues.
